src/test/java/training/article/responsearticles.json

```diff
@@ -0,0 +1,22 @@
+{
+  "articles": [
+    {
+      "title": "#string",
+      "slug": "#string",
+      "body": "#string",
+      "createdAt": "#string",
+      "updatedAt": "#string",
+      "description": "#string",
+      "tagList": [],
+      "author": {
+        "username": "#string",
+        "bio": "#string",
+        "image": "#string",
+        "following": "#boolean"
+      },
+      "favorited": "#boolean",
+      "favoritesCount": "#number"
+    }
+  ],
```

Comment:
this is exactly the same of the other json. I recommend you check if this `articles` is an array and after that check if every item of it is an article (that you defined in `responsesuccessfully.json`)